src/map/skill.c

```diff
@@ -3498,10 +3503,10 @@ static int skill_attack_copy_unknown(int *attack_type, struct block_list *src, s
 
 static int skill_attack_dir_unknown(int *attack_type, struct block_list *src, struct block_list *dsrc, struct block_list *bl, uint16 *skill_id, uint16 *skill_lv, int64 *tick, int *flag)
 {
-	return -1;
+	return UNIT_DIR_UNDEFINED;
 }
 
-static void skill_attack_blow_unknown(int *attack_type, struct block_list *src, struct block_list *dsrc, struct block_list *bl, uint16 *skill_id, uint16 *skill_lv, int64 *tick, int *flag, int *type, struct Damage *dmg, int64 *damage, int8 *dir)
+static void skill_attack_blow_unknown(int *attack_type, struct block_list *src, struct block_list *dsrc, struct block_list *bl, uint16 *skill_id, uint16 *skill_lv, int64 *tick, int *flag, int *type, struct Damage *dmg, int64 *damage, enum unit_dir *dir)
```

Comment:
Well, I didn't count it, but this line obviously is longer than 120 columns. Break it, please. :relaxed:

Reply:
I don't think that we do that with function headers.

Reply:
We do. Quote from style guide:

> Statements longer than 120 columns will be broken into sensible chunks, unless exceeding 120 columns significantly increases readability and does not hide information.
...
The same applies to function headers with a long argument list.

The reason behind this rule is to guarantee the readability even on terminals, so it would be pointless to exclude function headers.

Reply:
rip. okey okey